homeassistant/components/slide/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "slide",
```

Comment:
`slide` is too generic I think. Can we name it `goslide` instead? Our standard is to name integrations after the brand names.

Reply:
They are using 'slide' as a brand/product name (https://slide.store). It was initially goslide, but have rebranded everything now to slide. In theory this matches the guideline, but if you prefer to be named 'goslide', i will rebrand again.

Reply:
@balloob what do you think?

Reply:
Slide is fine. I doubt that we'll ever get smart slides integrated